src/Components/EventTab/EventTab.jsx

```diff
@@ -1,7 +1,98 @@
+import { useState } from "react";
 import styles from "./EventTab.module.scss";
+import EventCard from "../EventCard/EventCard";
+import DayData from "../../Data/EventDaysInfo.json";
+import EventCarousal from "../EventCarousal/EventCarousal";
 
 const EventTab = () => {
-  return <div className={styles.container}>.styles</div>;
+  const { day1, day2, day3 } = DayData;
+  const [dayId, setDay] = useState(day1);
+  const [barOne, setbarOne] = useState(styles.long);
+  const [barTwo, setbarTwo] = useState(styles.short);
+  const [barThree, setbarThree] = useState(styles.short);
+  const [activeBar, setActiveBar] = useState(styles.activeOne);
```

Comment:
you should combine these different data variables in a single object state

variables are not in the proper naming convention like `setbarOne` should be `setBarOne`